daemon/src/projection.rs

```diff
@@ -432,9 +497,28 @@ impl Cfd {
 
         // If we don't have a dedicated closing price, keep the one that is set (which is
         // based on current price).
-        if let Some(closing_price) = self.aggregated.closing_price {
-            let (profit_btc, profit_percent) = self.compute_profit(closing_price);
+        // TODO: This clone can proably be optimized away
+        if let Some(payout) = self.aggregated.clone().payout() {
+            let payout = payout
+                .to_signed()
+                .expect("Amount to fit into signed amount");
+
+            let (profit_btc, profit_percent) = match calculate_profit(
+                payout,
+                self.margin
+                    .to_signed()
+                    .expect("Amount to fit into signed amount"),
+            ) {
+                Ok((profit_btc, profit_percent)) => {
+                    (Some(profit_btc), Some(profit_percent.to_string()))
+                }
+                Err(err) => {
+                    tracing::error!(%payout, margin=%self.margin, "Profit calculation failed: {:#}", err);
+                    (None, None)
+                }
+            };
 
+            self.payout = Some(payout);
```

Comment:
I think we should be distinguishing between `realized` and `unrealized` profits and model it that way, e.g. call the fields `realized_profit` and `unrealized_profit`, `payout` and `predicted_payout` (or similar).

To decide whether to calculate `realized` or `unrealized` profits should depend on the state (a CFD is either `open` or `closed`). 
Right now the logic of how to calculate the profit (`realized` or `unrealized`) is spread into different places which I find hard to follow. 

We don't have to do this in this PR but I think the complexity here will bite us eventually.

Reply:
I totally agree, but that would be a breaking API change. I tried to avoid that and keep what we have but make it more correct that in was before. We can go further and change the API as well.

Reply:
> If we accept breaking the API then we could even go further. We could distinguish more clearly between values that are fixed and values that are dynamic (e.g. based on current price). We could more clearly model "unrealized" (= open) and "realized" (= closed).

If you don't remove the old field but just add two new ones, it is not a breaking change :)

Reply:
After some more thought I am actually not sure the current UI has an advantage in a split API. We would have to introduce decision logic for "is realized already set otherwise use unrealized". 
I still think we should change the API, but I don't think it should be done as part of this PR. We could potentially also use two separate UI components for open and closed positions in the UI. I feel this needs a bit more thinking and should not be rushed as part of this work.

Reply:
> I still think we should change the API, but I don't think it should be done as part of this PR.

this 👍